curriculum/js2/4.js

```diff
@@ -5,7 +5,8 @@
  */
 
 const solution = (a) => {
-  return 0;
+  if (!a.length) return '';
```

Comment:
if your code is to return sum of all values, result should be a number. so for edge / not applicable cases, returning a number (-1? 0?) is better than returning a string to keep types consistent